src/appkit.cs

```diff
@@ -21320,7 +21320,7 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 		NSImage IconForFiles (string[] fullPaths);
 		
 		[Export ("iconForFileType:"), ThreadSafe]
-		NSImage IconForFileType (string fileType);
+		NSImage IconForFileType (IntPtr fileTypeOrTypeCode);
```

Comment:
Since you added better API let's mark this one with `[Internal]` since its usage is not clear